LinearRegression/LinearRegressionWPF/BackendDescriptors/AvailableModels.cs

```diff
@@ -29,28 +31,28 @@ static AvailableModels()
                 }
             };
 
-            AvailableOptimizers = new Dictionary<string, OptimizerDescriptor> {
+            AvailableOptimizers = new Dictionary<string, EstimatorDescriptor> {
                 {
                     "SimpleOrdinaryLeastSquare",
-                    new OptimizerDescriptor {
+                    new EstimatorDescriptor {
                         Name = "Simple Ordinary Least Square",
                         IsIterative = false,
-                        SupportedParameters = new OptimizerBuilderParams.Parameter[] { },
+                        SupportedParameters = Array.Empty<OptimizerBuilderParams.Parameter>(),
                         BuildOptimizer = (parameters) => new SimpleOrdinaryLeastSquare()
                     }
                 },
                 {
                     "QuadraticOrdinaryLeastSquare",
-                    new OptimizerDescriptor {
+                    new EstimatorDescriptor {
                         Name = "Quadratic Ordinary Least Square",
                         IsIterative = false,
-                        SupportedParameters = new OptimizerBuilderParams.Parameter[] { },
+                        SupportedParameters = Array.Empty<OptimizerBuilderParams.Parameter>(),
                         BuildOptimizer = (parameters) => new QuadraticOrdinaryLeastSquare()
                     }
                 },
                 {
                     "GradientDescent",
```

Comment:
The Gradient Descent is optimizer (because it is iterative)

Reply:
Oh okay, now I realized this is a problem, because all the commands and such expect an optimizer. I thought you just renamed optimizer to estimator, but now I see the distinction. But this is pretty hard to solve, because we would need a seperate dropdown menu for estimators, and also make sure only one estimator or one optimizer can be selected at a time. Or we could create a common type for both, but then why differetiate them in the first place.